kai/reactive_codeplanner/task_manager/api.py

```diff
@@ -15,7 +15,8 @@ class RpcClientConfig:
     analyzer_java_bundle_path: Path
     label_selector: Optional[str]
     incident_selector: Optional[str]
-    included_paths: Optional[list[str]]
+    included_paths: Optional[list[str]] = None
+    excluded_paths: Optional[list[Path]] = None
```

Comment:
Should we update `included_paths` to be `list[Path]` as opposed to `list[str]`?

Reply:
Should we? jk...on it